CHANGELOG-gui.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog].
 ### Added
 
 ### Changed
+- Updated French Translation [`#1442`](https://github.com/vrc-get/vrc-get/pull/1494)
```

Comment:
localization changes doesn't need changelog. (CI was confused because helper script is added(

Reply:
I reverted the commit.

Reply:
Will potentially do a separate MR for the script